tests/arm-tv/ptrroundtrips/round-trip-test-0.aarch64.ll

```diff
@@ -0,0 +1,10 @@
+; TEST-ARGS: -run-replace-ptrtoint
+; should collapse into gep
+define ptr @f(ptr %p, i64 %idx) {
+    %i1 = ptrtoint ptr %p to i64
+    %i2 = add i64 %i1, %idx
+    %r = inttoptr i64 %i2 to ptr
+    ret ptr %r
+}
+
+; CHECK: gep ptr %#0
```

Comment:
see if you can tighten up the check a bit, like add a CHECK-NOT for ptrtoint and inttoptr